lms/static/certificates/sass/_components.scss

```diff
@@ -0,0 +1,641 @@
+}
+
+// ------------------------------
+// #OVERRIDE - BUTTONS
+// ------------------------------
+a.btn {
+
+    &:hover, &:active, &:focus {
+        border-bottom-color: $transparent;
+    }
+}
+
+.btn-default {
+    background: $transparent;
+
+    // STATE: hover and focus
+    &:hover,
+    &.is-hovered,
+    &:focus,
+    &.is-focused {
+        background: palette(primary, base);
+        color: palette(grayscale, white);
+    }
+}
+
+// ------------------------------
+// #DEPTH
+// ------------------------------
+%depth-base-focus {
+    background: palette(grayscale, white-t);
+    padding: spacing-vertical(small) spacing-horizontal(base);
+}
+
+%depth-well {
+    box-shadow: 0 rem(1) rem(3) 0 palette(grayscale-cool, light) inset;
+    padding: spacing-vertical(small) spacing-horizontal(base);
+    background: palette(grayscale, x-light);
+}
+
+%depth-card {
+    box-shadow: 0 rem(1) rem(2) 0 palette(grayscale-cool, light);
+    padding: spacing-vertical(small) spacing-horizontal(base);
+    background: palette(grayscale, white-t);
+}
+
+.depth-base-focus {
+    @extend %depth-base-focus;
+}
+
+.depth-well {
+    @extend %depth-well;
+}
+
+.depth-card {
+    @extend %depth-card;
+}
+
+
+// ------------------------------
+// #IMAGES
+// ------------------------------
+// circular cropped
+%img-cropped {
+
+    .src {
+        display: block;
+        width: 100%;
+        border-radius: 50%;
+    }
+}
+
+// ------------------------------
+// #HEADER APP
+// ------------------------------
+.wrapper-header {
+    @extend %divider-2;
+    border-bottom-color: palette(grayscale, xx-trans);
+    background: palette(grayscale, white-t);
+}
+
+.header-app {
+
+    .logo {
+        @extend %reset-link;
+    }
+}
+
+// ------------------------------
+// #BANNER
+// ------------------------------
+.banner {
+
+    .message-copy {
+        margin-bottom: 0;
+    }
+}
+
+// message to user-centric banner
+.wrapper-banner-user {
+    box-shadow: inset 0 rem(2) rem(2) 0 palette(grayscale, black-t);
+    background: palette(grayscale-cool, x-dark);
+}
+
+.banner-user {
+
+    .message-title {
+        color: palette(grayscale, white-t);
+        font-weight: font-weight(semi-bold);
+    }
+
+    .message-copy {
+        color: palette(grayscale, white);
+    }
+
+    .message-actions .action {
+        display: inline-block;
+        vertical-align: middle;
+        margin-right: spacing-horizontal(small);
+        font-size: font-size(mid-large);
+
+        &:last-child {
+            margin-right: 0;
+        }
+    }
+}
+
+// ------------------------------
+// #INTRODUCTION
+// ------------------------------
+.introduction {
+    margin-bottom: spacing-vertical(base);
+    text-align: center;
+}
+
+// ------------------------------
+// #ACCOMPLISHMENTS
+// ------------------------------
+// base accomplishment
+.accomplishment {
+
+    .accomplishment-type-symbol {
+        @extend %img-cropped;
+    }
+}
+
+// main accomplishment
```

Comment:
NOTE: considering a few layers of basic accomplishments with this set up - a main accomplishment (the focus of a view) vs. brief accomplishments (supplementally listed or listed in contexts of other application views - e.g. profile, progress, dashboard).